config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
```

Comment:
This needs to be changed so that the configuration is stored in `app/models/configuration.rb` rather than Rails.configuration